SecretSanta/src/SecretSanta.Web/SecretSanta.Web.csproj

```
@@ -3,6 +3,11 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="3.1.1" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="3.1.1" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="3.1.1" />
+  </ItemGroup>>
```

```suggestion
  </ItemGroup>
```
1>Build FAILED.
1>
1>...\SecretSanta\src\SecretSanta.Web\SecretSanta.Web.csproj(1,1): error MSB4067: The element <#text> beneath element <Project> is unrecognized.
1>    0 Warning(s)
1>    1 Error(s)
1>
1>Time Elapsed 00:00:00.05